src/Solvers/WordSearchComponent.tsx

```diff
@@ -0,0 +1,171 @@
+import { WordSearchSolver, WordSearchResult } from 'puzzle-lib';
+import React, { FormEvent, MouseEvent, SyntheticEvent } from 'react';
+import { Button, ButtonGroup, ButtonToolbar, FormControl, Grid, Row, Col } from 'react-bootstrap';
+import LocalStorageComponent from '../Data/LocalStorageComponent';
+import './WordSearchComponent.css';
+
+type Props = {};
+type State = {
+  gridInputText: string,
+  wordListInputText: string,
+  output: JSX.Element[],

```

Comment:
I would just render on the fly rather than storing elements in the output.  I suspect there are React runtime warnings to suggest the same thing.

Reply:
There weren't for this.  Saw some for keys, and fixed those.